src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  const ERROR_MESSAGE = 'Bzzz... Error!';
+  let counter = 0;
+
+  return function getSum(x) {
+    counter++;
+
+    const isEvenCall = counter % 2 === 0;
+    const isGreaterThenThree = counter > 3;
```

Comment:
I think you could just combine these two conditions into one variable (e.g. isError)

Reply:
I have two of them so it would be more descriptive what exactly are we checking